### PR TITLE
Fixes #8717: Add support for CFEngine developpment environment to rtf

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/8717

### CODE REVIEW

**Comment by amousset** [comment]
wip

File: scripts/cfengine.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Tested on CentOS 7
+
+yum install -y patch git epel-release
+# Build dependencies
+yum install -y gcc lmdb-devel openssl-devel pcre-devel pam-devel libtool bison flex
```
Comment (amousset) [inline scripts/cfengine.sh]: + libxml2-devel

File: scripts/cfengine.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+# Development dependencies
+yum install -y gdb valgrind
+# Clone repo and build with options similar to Rudder packaging
+su vagrant -c "cd && git clone https://github.com/cfengine/core.git && cd core \
+                  && ./autogen.sh --prefix=/opt/rudder --with-workdir=/var/rudder/cfengine-community --enable-static=yes --enable-shared=no \
```
Comment (amousset) [inline scripts/cfengine.sh]: --enable-debug

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
WIP

**Comment by fanf** [comment]
As this is a long-running WIP, I'm closing the PR. Alexis branch can be followed to see evolution, and a new PR will be done when more work is achieved.

**Pull request closed without merging**